ingest/log/logging.go

```diff
@@ -372,18 +373,18 @@ func (l *Logger) ErrorfWithDepth(d int, f string, args ...interface{}) error {
 	return l.outputf(d, ERROR, f, args...)
 }
 
-// CriticalfWithDepth writes a CRITICALinfo level log to the underlying writer,
+// CriticalfWithDepthf writes a CRITICALinfo level log to the underlying writer,
```

Comment:
Although this is description wasn't changed, it is exactly the same as the description for `Criticalf` above. It should probably mention the way in which `CriticalfWithDepthf` is different.